packages/sdk/src/libs/clients/emp/client.ts

```diff
@@ -1,11 +1,13 @@
-import { ExpiringMultiParty__factory, ExpiringMultiParty } from "@uma/core/contract-types/ethers";
+import { EthersContracts } from "@uma/core";
 import type { SignerOrProvider, GetEventType } from "../..";
 import { Event } from "ethers";
 import { Balances } from "../../utils";
 
-export type Instance = ExpiringMultiParty;
+export type Instance = EthersContracts.ExpiringMultiParty;
+const Factory = EthersContracts.ExpiringMultiParty__factory;
```

Comment:
@mrice32 i know you wont like this 😀 but having common naming between all client files makes it easy to 1. template new clients and 2. refactor across all clients, like with this pr

Reply:
haha, the reason I don't like it is that it seems like it's designed to make copy and pasting easier, but then if the code is exactly the same, why can't it just be a generic?

Either way, I'm not strongly opinionated :)

Side note: are these cosmetic changes related to the compilation issues? Or are they just purely cosmetic?

Reply:
thats a good point, i didnt think of that. you might be on to something

Reply:
also these are mainly cosmetic changes. the fix really was just with changing the imports